crates/index-scheduler/src/batch.rs

```diff
@@ -1232,6 +1235,44 @@ impl IndexScheduler {
         index: &'i Index,
         operation: IndexOperation,
     ) -> Result<Vec<Task>> {
+        let indexer_alloc = Bump::new();
+
+        let started_processing_at = std::time::Instant::now();
+        let secs_since_started_processing_at = AtomicU64::new(0);
+        const PRINT_SECS_DELTA: u64 = 1;
```

Comment:
```suggestion
        const PRINT_SECS_DELTA: u64 = 5;
```

We don't need a progress log every second